src/pages/Player.vue

```diff
@@ -401,6 +482,34 @@ export default {
   },
   methods: {
     ...mapActions("auth", ["setAccessToken", "setActiveWorkspace"]),
+    calculateScorecardMetrics() {
+      // calculate scorecard metrics using `itemResponses`
```

Comment:
in past comments, itemResponses and other variables have been written without ` - let's stay consistent.

```suggestion
      // calculate scorecard metrics using itemResponses
```